src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -328,6 +339,27 @@ void IgnRenderer::Render(RenderSync *_renderSync)
   // update and render to texture
   this->dataPtr->camera->Update();
 
+  if (!this->cameraViewController.empty())
+  {
+    std::string viewControlService = "/gui/camera/view_control";
+    transport::Node node;
+    std::function<void(const msgs::Boolean &, const bool)> cb =
+        [&](const msgs::Boolean &/*_rep*/, const bool _result)
+    {
+      if (!_result)
+        ignerr << "Error setting view controller. Check if view angle GUI "
+                  "plugin is already loaded" << std::endl;
+      else
+      {
+        this->cameraViewController = "";
```

Comment:
Don't we always want to set this back to an empty string regardless if `_result` is true or false? If `_result == false`, then the request will be sent at each `Render()` call